app/controllers/posts_controller.rb

```diff
@@ -9,4 +9,30 @@ def show
     @post = Post.find(params[:id])
     @comments = @post.comments
   end
+
+  def new
+    post = Post.new
+    respond_to do |format|
+      format.html { render :new, locals: { post: post } }
+    end
+  end
+
+  def create
+    post = Post.new(params.require(:post).permit(:title, :text))
```

Comment:
- [ ] <b>REQUIRED:</b> The best way to create a post as it is required to use `current_user` in this milestone is by using curent_user as follow: 

```ruby
@post = @current_user.posts.new(post_params)
```

And as you might remember in this [lesson](`https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides`). The best and secured way of passing parameter is as follow:

```ruby
    private 
    def post_params
      params.require(:post).permit(:title, :text)
    end
```